ical Example

ger82 opened the pull request:
So nun hab ich mir nochmal Zeit genommen.

Ich hoffe nun passt es.

Commit:
ical Example

Commit:
Update ical_calendar.php

- Zeitzonen Anpassung
- englische Variablen

jdede commented:
Moin und vielen Dank nochmal für deinen Beitrag. Ich habe es gerade mal nachvollzogen und bei mir getestet. Ich habe noch ein paar kleine Anmerkungen:

- Du verwendest https://github.com/jamct/DoorsignEPD/blob/039e40cbe87bea16f42ae8027caf417b05a346f8/server/contents/ical_calendar.php#L19 
Jedoch ist der Standardpfad für die Installation via composer (wenn ich mich nicht irre) `vendor/autoload.php`. Kannst du die php-Datei bitte noch entsprechend anpassen?
- Die Demo ist für ein 7-Zoll-Display, Hochformat. Vielleicht noch einen Hinweis darauf in der Readme?
- Die Demo-ics-Datei enthält Termine für Ende Februar / Anfang März, d.h. in einer Woche wäre der Kalender leer. Woher stammt die Kalender-Datei? Kann man in der Readme da evtl. noch drauf verweisen bzw. der Nutzer eine neue Demo-Datei herunterladen?

Danke

ger82 commented:
Den vendor Pfad habe ich angepasst und die Readme erweitert.
Der Übersichtlichkeit halber ist es meiner Meinung nach dennoch sinnvoll vobject in einen separaten Ordner abzulegen.
Der Kalender stammt von mir, er enthält 2 Termine die sich wöchendlich wiederholen, ohne definiertes Enddatum. Direkt herunterladen geht leider nicht da der Owncloud Server lokal läuft.

Commit:
ical fixes

- vendor Pfad angepasst
- link zu altem ical Kalender von evert entfernt
- readme ergänzt

jdede commented:
>  Der Übersichtlichkeit halber ist es meiner Meinung nach dennoch sinnvoll vobject in einen separaten Ordner abzulegen.

Grundsätzlich schon, jedoch installiert composer die benötigten Bibliotheken schon nach `vendor/sabre/vobject`. Dort liegen auch noch weitere Abhängigkeiten wie z.B. `uri` und `xml`. `vendor/autoload.php` bindet dieses dann ein. Vendor ist also schon ein solches Verzeichnis, welches die Bibliotheken enthält.

Ich teste es später noch mal und wenn es keine Probleme gibt, merge ich das in den Master.
